settings.gradle

```diff
@@ -24,7 +24,7 @@ buildCache {
     remote(develocity.buildCache) {
         enabled = true
         // Check access key presence to avoid build cache errors on PR builds when access key is not present
-        def accessKey = System.getenv("GRADLE_ENTERPRISE_ACCESS_KEY")
+        def accessKey = System.getenv("DEVELOCITY_ACCESS_KEY")
```

Comment:
@erichaagdev is this env var made available by the gradle setup action?

Reply:
It will also look for the `DEVELOCITY_ACCESS_KEY` environment variable if supplied as an explicit `env`, create a short-lived access token, then replace your provided `DEVELOCITY_ACCESS_KEY` with the short-lived access token.

This is effectively how all the CI integrations work IIRC.

Reply:
Thanks for the explanation.